makejdk.sh

```diff
@@ -197,24 +239,15 @@ buildAndTestOpenJDKViaDocker()
   fi
 
   echo "${info}Using Docker to build the JDK${normal}"
+  
+  createPersistentDockerDataVolume
 
-  CONTAINER=openjdk_container
-
-  # Copy our script for usage inside of the container
-  rm docker/jdk8u/x86_64/ubuntu/colour-codes.sh
-  cp "${SCRIPT_DIR}/sbin/colour-codes.sh" docker/jdk8u/x86_64/ubuntu 2>/dev/null
 
-  rm docker/jdk8u/x86_64/ubuntu/build.sh
-  cp "${SCRIPT_DIR}/sbin/build.sh" docker/jdk8u/x86_64/ubuntu 2>/dev/null
+  # Copy our scripts for usage inside of the container
+  rm -r docker/jdk8u/x86_64/ubuntu/sbin
```

Comment:
jdk8u probably needs to be parameterised later, but that can be a future PR I guess.